domain/src/main/java/com/astasko/pdp/domain/model/User.java

```diff
@@ -0,0 +1,94 @@
+package com.astasko.pdp.domain.model;
+
+import com.astasko.pdp.domain.model.common.BaseEntity;
+import com.astasko.pdp.domain.model.common.StoredEntity;
+
+public class User extends BaseEntity<Long> {
```

Comment:
Add equals, hashCode, and toString methods (use apache-commons for that)

Reply:
Added